MXCHIP/mxchip_basic/mxchip_basic.ino

```diff
@@ -30,7 +35,8 @@ static IOTContext context = NULL;
 
 static bool isConnected = false;
 
-void onEvent(IOTContext ctx, IOTCallbackInfo *callbackInfo) {
+void onEvent(IOTContext ctx, IOTCallbackInfo *callbackInfo)
+{
```

Comment:
Why?

Reply:
good catch, will revert to maintain the common style